src/transformers/__init__.py

```diff
@@ -260,6 +260,7 @@
         "ERNIE_PRETRAINED_CONFIG_ARCHIVE_MAP",
         "ErnieConfig",
     ],
+    "models.ernie_m": ["ERNIE_M_PRETRAINED_CONFIG_ARCHIVE_MAP", "ErnieMConfig", "ErnieMTokenizer"],
```

Comment:
The tokenizer requires sentencepice, so its import should be protected.

Reply:
I am sorry, I understood your previous comment - `The tokenization auto file should also be updated with the new tokenizer.` but didn't get this one(`The tokenizer requires sentencepice, so its import should be protected.`) could you please elaborate it?

Reply:
The tokenizer should only be in the `if is_sentencepiece_available` part of the init.